kfs-core/src/main/java/edu/arizona/kfs/fp/document/web/struts/DisbursementVoucherForm.java

```diff
@@ -0,0 +1,37 @@
+package edu.arizona.kfs.fp.document.web.struts;
+
+import edu.arizona.kfs.fp.document.DisbursementVoucherDocument;
+import edu.arizona.kfs.sys.KFSConstants;
+
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.coreservice.framework.parameter.ParameterService;
+import org.kuali.rice.kns.service.DataDictionaryService;
```

Comment:
Deprecated class, should be using org.kuali.rice.krad.service.impl.DataDictionaryService